x-pack/plugins/security_solution/cypress/e2e/dashboards/entity_analytics.cy.ts

```diff
@@ -45,14 +49,73 @@ describe('Entity Analytics Dashboard', () => {
     });
   });
 
+  describe('Host Risk Score enabled but still no data', () => {
```

Comment:
NIT: You can group the tests that are checking more or less the same to make the execution faster:

i.e.
describe('Host Risk Score and User Risk Score enabled but still no data', () => {
    before(() => {
      esArchiverLoad('risk_hosts_no_data');
      esArchiverLoad('risk_users_no_data');
      visit(ENTITY_ANALYTICS_URL);
    });

    after(() => {
      esArchiverUnload('risk_hosts_no_data');
      esArchiverUnload('risk_users_no_data');
    });

    it('shows no data detected prompt for the hosts', () => {
      cy.get(HOST_RISK_SCORE_NO_DATA_DETECTED).should('be.visible');
    });
    
     it('shows no data detected prompt for the users', () => {
      cy.get(USER_RISK_SCORE_NO_DATA_DETECTED).should('be.visible');
    });
  });

   
In this way, you minimize the times you visit the page since is an action that takes time to be executed. Following the same pattern, you can group the legacy data tests.